platforms/jvm/testing-jvm/src/test/groovy/org/gradle/api/tasks/testing/TestTaskSpec.groovy

```diff
@@ -323,7 +323,7 @@ class TestTaskSpec extends AbstractProjectBuilderSpec {
         given:
         expectFirstTestSuiteContainsTestButSecondIsEmpty()
         task.filter {
-            it.includePatterns = "Foo"
+            it.includePatterns = ["Foo"]
```

Comment:
🤔 Ah, that's another magic Groovy conversion, right? `T` -> `Set<T>`. Should we add this as a deprecated feature?

Reply:
I guess so yeah. Added it to the list: https://github.com/gradle/gradle/issues/31372